draft-ietf-teas-actn-poi-applicability.txt

```diff
@@ -2492,6 +2546,15 @@ Internet-Draft                 ACTN POI                    January 2023
    bundled-link container defined in [RFC8795]) bundling the multi-layer
    single-domain Ethernet link(s) being created, as described above.
 
+   When a new LAG link is created, it is also recommended to configure
+   the minimum number of active member links required to consider the
+   LAG link as being up. For example, a LAG link with three members can
+   be considered up when only one member link fails and down when at
+   least two member links fail.
+
+   This configuration is missing in [CLIENT-TOPO] and this is identified
+   as a gap in section 6.
```

Comment:
Just to be clear: the model is read only and the intention is to extend that with write part for configuration?

Reply:
**2023-07-04 TEAS Call**

Agreed to clarify that the gap is about a missing attribute and not the whole configuration of LAG

Reply:
Fixed in https://github.com/IETF-TEAS-WG/actn-poi/pull/114/commits/65ec90d608f752d341a829c66e428d418c45da90